lib/image-stream/README.md

```diff
@@ -58,4 +58,19 @@ These are the rules for handling archive images:
 
 The module throws an error if the above rules are not met.
 
+Supported Formats
+-----------------
+
+There are currently three image types in supported formats: `image`, `compressed` and `archive`.
+
+An extension tagged `image` describes a format which can be directly written to a device by its handler,
+and an extension tagged `archive` denotes an archive containing an image, and will cause an archive handler
+to open the archive and search for an image file.
```

Comment:
Nope, because Etcher only checks one level deep in archives, and yells if it can't find a suitable image